int32   fps
int32   ball_x
int32   ball_y
string  ball_LR
int32   ball_ang
int32   ball_dis

int32   blue_x
int32   blue_y
string  blue_LR
int32   blue_ang
int32   blue_dis
int32   blue_fix_x
int32   blue_fix_y
int32   blue_fix_ang
int32   blue_fix_dis

int32   yellow_x
int32   yellow_y
string  yellow_LR
int32   yellow_ang
int32   yellow_dis
int32   yellow_fix_x
int32   yellow_fix_y
int32   yellow_fix_ang
int32   yellow_fix_dis
